### PR TITLE
Create expo wrapper module

### PR DESCRIPTION
**Pull request by vishnumad**

### CODE REVIEW

File: react-native/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# coinbase-wallet-sdk-expo
+
+Coinbase Wallet Native SDK
```
Comment (bangtoven) [inline react-native/README.md]: nit:
```suggestion
Coinbase Wallet SDK
```
or
```suggestion
Coinbase Wallet Mobile SDK
```

**Review by bangtoven — DISMISSED** [review]
awesome!!!

File: react-native/android/build.gradle [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+// Creating sources with comments
+task androidSourcesJar(type: Jar) {
+  classifier = 'sources'
+  from android.sourceSets.main.java.srcDirs
+}
+
+afterEvaluate {
+  publishing {
+    publications {
+      release(MavenPublication) {
```
Comment (AMITGOELNY) [inline react-native/android/build.gradle]: are we going to publish via maven. If not, we can likely remove.

File: react-native/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@coinbase/wallet-mobile-sdk-expo",
```
Comment (bangtoven) [inline react-native/package.json]: is it a convention to have `-expo` suffix ?

File: react-native/android/build.gradle [hunk 4]
```diff
@@ -0,0 +1,94 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'maven-publish'
+
+group = 'expo.modules.coinbasewalletsdkexpo'
```
Comment (AMITGOELNY) [inline react-native/android/build.gradle]: can likely remove group and version here, if not publishing to maven

File: react-native/ios/CoinbaseWalletSDKExpo.podspec [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  s.license        = package['license']
+  s.author         = package['author']
+  s.homepage       = package['homepage']
+  s.platform       = :ios, '12.0'
+  s.swift_version  = '5.4'
+  s.source         = { git: 'https://github.com/coinbase/wallet-mobile-sdk' }
+  s.static_framework = true
+
+  s.dependency 'ExpoModulesCore'
+  s.dependency 'CoinbaseWalletSDK', '0.1.2'
```
Comment (bangtoven) [inline react-native/ios/CoinbaseWalletSDKExpo.podspec:19]: TODO: remove version requirement once ios and android have parity

**Review by bangtoven — DISMISSED** [review]

File: react-native/package.json (hunk 3 above)
Comment (vishnumad) [reply react-native/package.json]: Not sure, I can remove it

**Review by AMITGOELNY — APPROVED** [review]

**Pull request merged**